sos/plugins/pcp.py

```diff
@@ -136,6 +136,16 @@ def setup(self):
 
         # Need to get the current status of the PCP infrastructure
         self.add_cmd_output("pcp")
-
+        # Collect a summary for the current day
+        res = self.get_command_output('pcp')
```

Comment:
Can't be merged with `self.add_cmd_output("pcp")`` few lines above into one call?